src/DI/GoSmsExtension.php

```diff
@@ -31,27 +31,56 @@ public function getConfigSchema(): Nette\Schema\Schema
 
 	public function loadConfiguration(): void
```

Comment:
Tohle rozdeleni do metod mi osobne neprijde vic prehledne nez to bylo. Slo by to vratit?

Reply:
ok